scripts/install/git.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+git config --global diff.renameLimit 200000
+git config --global credential.helper cache
+git config --global credential.helper 'cache --timeout=9999999999999'
```

Comment:
We should not be using the `cache` helper with a large timeout. There is a [`store` credential helper](https://git-scm.com/docs/git-credential-store) for this exact use case, please use that one instead.

Reply:
I made this change @ccarterlandis.